main.js

```diff
@@ -190,6 +238,9 @@ define(function (require, exports, module) {
     function loadPreferences() {
         _preferences = PreferencesManager.getExtensionPrefs(preferencesId);
         _preferences.definePreference("checked", "boolean", defaultPreferences["checked"]);
+        _preferences.definePreference("colors", "Object", defaultPreferences["colors"]);
+        _preferences.on("change", updateColors);
+        PreferencesManager.getExtensionPrefs("themes").on("change", updateColors);
```

Comment:
These listeners should be moved to another function, which executes _after_ an individual call to `updateColors()`. In this way, the styles are applied more than once on app load.